server/users/users.factory.ts

```diff
@@ -0,0 +1,68 @@
+import { Sports } from "../../database/models/sports"
+import { Users } from "../../database/models/users"
+import ValidationException from "../core/exceptions/validation.exception"
+import { SportModel } from "../sports/sports.model"
+import { UserSettingsModel } from "./user-settings.model"
+import { SportSubscriptionsModel } from "./user-sport-subscriptions.model"
+import { UserModel } from "./users.model"
+
+export class UserFactory {
+
+	public static Create(email: string, password: string, isAdmin: boolean = false) {
+
+        const defaultUserSettings = new UserSettingsModel(undefined, undefined)
+
+		return new UserModel(
+			undefined,
+			email,
+			password,
+			isAdmin,
+			[],
+			defaultUserSettings
+		)
+	}
+
+	public static FromDatabase(
+		user: Users,
+        sports: SportModel[] | undefined = undefined,
```

Comment:
This probably won't be possible to be `undefined` when we stop using it in `AuthService.`

Reply:
Resolved in: https://github.com/mbrizic/ratethegame/pull/28/commits/46acfffc8658ade37627b61334f80ce4b6ab3fb4